### PR TITLE
familiar to formal

### PR DESCRIPTION
**Pull request by jnethe**
from the familiar case to formal

### CODE REVIEW

**Comment by vypr** [comment]
Thanks again, @jnethe!

File: i18n/spanish.json [hunk 1]
```diff
@@ -18,10 +18,10 @@
     "PGTXT_SIGNUP_PASSWORD":"Contraseña:",
     "PGTXT_SIGNUP_PASSWORD_PLC":"Contraseña",
     "PGTXT_SIGNUP_CONPASSWORD":"Confirma la contraseña:",
-    "PGTXT_SIGNUP_CONPASSWORD_PLC":"Confirma tu contraseña",
+    "PGTXT_SIGNUP_CONPASSWORD_PLC":"Confirma su contraseña",
```
Comment (anmrdz) [inline i18n/spanish.json]: The informal way is : "Confirma tu contraseña"
The formal way is: "Confirme su contraseña", this "confirma su contraseña" is mixing informal and formal

File: i18n/spanish.json [hunk 2]
```diff
@@ -18,10 +18,10 @@
...
     "PGTXT_SIGNUP_EMAIL":"Correo:",
     "PGTXT_SIGNUP_EMAIL_PLC":"Correo",
-    "PGTXT_SIGNUP_CONEMAIL":"Confirmar correo:",
+    "PGTXT_SIGNUP_CONEMAIL":"Confirma correo:",
```
Comment (anmrdz) [inline i18n/spanish.json]: The same thing this should be "Confirme su correo".

File: i18n/spanish.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "RSP_SIGNUP_EMAIL_USED":"Este correo ya existe en nuestra base de datos, contáctate con soporte si necesitas recuperar tu cuenta.",
     "RSP_SIGNUP_USER_NOWHTSPC":"Lo sentimos, pero los nombres de usuario no pueden contener espacios.",
     "RSP_SIGNUP_INVALID_EMAIL":"Por favor utiliza un correo valido.",
-    "RSP_SIGNUP_INVALID_CAPTCHA":"No te puedes registrar hasta que realices el CAPTCHA correctamente.",
+    "RSP_SIGNUP_INVALID_CAPTCHA":"No se puedes registrar hasta que realices el CAPTCHA correctamente.",
```
Comment (anmrdz) [inline i18n/spanish.json]: The formal way is: "No se puede registrar hasta que realice el CAPTCHA correctamente."

File: i18n/spanish.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "PG_PROFILE":"Perfil de %s",
 
     "PGTXT_HOME_GREETING":"Hola",
-    "PGTXT_HOME_LOGOUT_MSG":"Te gustaría",
+    "PGTXT_HOME_LOGOUT_MSG":"Se gustaría",
```
Comment (anmrdz) [inline i18n/spanish.json]: This is asking the user what he/she would want to do, in informal way is "Te gustaría," and in formal way is "Le gustaría", this "Se gustaría" is making reference of something that the software wants to do, not the user.

**Comment by anmrdz** [comment]
I did some comments on the translations, @jnethe please review them.

**Pull request merged**